CHANGELOG.md

```diff
@@ -29,6 +29,10 @@
 - Use `nfcore/gitpod:dev` container in the dev branch ([#2196](https://github.com/nf-core/tools/pull/2196))
 - Replace requests_mock with responses in test mocks ([#2165](https://github.com/nf-core/tools/pull/2165)).
 
+### Bug fixes, maintenance and tests
+
+- Add tests for `nf-core subworkflows create-test-yml` ([#2219](https://github.com/nf-core/tools/pull/2219))
+
```

Comment:
You add a new command under subworkflow and I think the changelog entry for it should live in the Subworkflows section.